sites/public/src/pages/sign-in.tsx

```diff
@@ -129,7 +134,12 @@ const SignIn = () => {
         await singleUseCodeFlow(email, true)
       }
       const { status } = error.response || {}
-      determineNetworkError(status, error)
+      const responseMessage = axios.isAxiosError(error) ? error.response?.data.message : ""
+      if (status === 400 && responseMessage?.includes("has not accepted the terms of service")) {
```

Comment:
Do we need both the 400 check and the response message check?

Reply:
This is what Morgan asked for on his previous review to match how we check the errors in determineStatusError

Reply:
I read Morgan's comments as not wanting to only check the 400, because that could catch multiple errors. I think if we just check the message we should be good to go, but I'm fine to check both, we just don't have to if we check the message.

Reply:
I think it is good to emulate what we have in determineStatusError, can't hurt